packages/supabase_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: supabase_flutter
 description: Flutter integration for Supabase. This package makes it simple for developers to build secure and scalable products.
-version: 2.0.0-dev.1
+version: 2.0.0-dev.2
```

Comment:
This one as well will automatically be updated during package release, so no need to change it now.